RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitOOTECAL_cff.py

```diff
@@ -1,6 +1,34 @@
 import FWCore.ParameterSet.Config as cms
 from RecoParticleFlow.PFClusterProducer.particleFlowRecHitECAL_cfi import *
 
-particleFlowOOTRecHitECAL = particleFlowRecHitECAL.clone()
-particleFlowOOTRecHitECAL.producers[0].qualityTests[1].timingCleaning = False
-particleFlowOOTRecHitECAL.producers[1].qualityTests[1].timingCleaning = False
+particleFlowRecHitOOTECAL = particleFlowRecHitECAL.clone()
+particleFlowRecHitOOTECAL.producers[0].qualityTests[1].timingCleaning = False
+particleFlowRecHitOOTECAL.producers[1].qualityTests[1].timingCleaning = False
+
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+
+## EB
+run2_miniAOD_80XLegacy.toModify(
+    particleFlowRecHitOOTECAL.producers[0], 
+    src = "reducedEcalRecHitsEB",
+    srFlags = ""
+)
+run2_miniAOD_80XLegacy.toModify(
+    particleFlowRecHitOOTECAL.producers[0].qualityTests[0], 
+    name = "PFRecHitQTestThreshold",
+    threshold = cms.double(0.08),
```

Comment:
in RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitECAL_cfi.py
```python
                  name = cms.string("PFRecHitQTestECALMultiThreshold"),
                  thresholds = particle_flow_zero_suppression_ECAL.thresholds
```
where particle_flow_zero_suppression_ECAL is from RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py and its default value for 2016 and earlier is the thresholds that you need (they are the same as here: EB at 80 MeV and EE at 300 MeV

Hence, these modifications are not necessary in this \*OOT\* file.